sklearn/discriminant_analysis.py

```diff
@@ -174,6 +175,25 @@ def _class_cov(X, y, priors, shrinkage=None, covariance_estimator=None):
     return cov
 
 
+def calc_posterior_proba(values):
+    """Compute posterior probability from log posterior.
+
+    Parameters
+    ----------
+    values : ndarray of shape (n_samples, n_classes)
+        Array of log posterior probabilities.
+
+    Returns
+    -------
+    post_prob : ndarray of shape (n_samples, n_classes)
+        Array of posterior probabilities.
+    """
+    likelihood = np.exp(values - values.max(axis=1)[:, np.newaxis])
+    # compute posterior probabilities
+    post_prob = likelihood / likelihood.sum(axis=1)[:, np.newaxis]
+    return post_prob
```

Comment:
```suggestion
def _from_discriminant_score_to_probability(discriminant_scores):
    """Compute posterior probabilities from discriminant scores.

    Parameters
    ----------
    discriminant_scores : ndarray of shape (n_samples, n_classes)
        Array of discriminant scores.

    Returns
    -------
    posterior_probailities : ndarray of shape (n_samples, n_classes)
        Array of posterior probabilities.
    """
    likelihood = np.exp(values - values.max(axis=1)[:, np.newaxis])
    # compute posterior probabilities
    return likelihood / likelihood.sum(axis=1)[:, np.newaxis]
```

Reply:
Wasn't sure whether to use single leading underscore since I'm calling it from outside of this module but much better function name.